R/factorize.R

```diff
@@ -9,6 +9,7 @@
 #' @param object a \code{secuTrialdata} object
 #' @param ... further parameters
 #' @export
+#' @return factorized \code{secuTrialdata} object
```

Comment:
I would say "\code{secuTrialdata} object with additional factor variables in any form containing factors"

Reply:
What does "in any form" exactly mean? Do you mean all the tables in the `secuTrialdata` object?

Reply:
the tables/data.frames represent forms, don't they... ;)

Reply:
We also have the meta data tables. These are not all forms.